### PR TITLE
AT-6348-form-field-page

### PR DESCRIPTION
**Pull request by tonysumner-ccpo**
Add the following form fields on a page

1. dropdown list
2. success textbox
3. error textbox
4. small collection of icons

All text present should be the ‘Source Sans Pro' font.

An additional library or js reference will be necessary to ensure the elements are properly colored in IE11, as Vuetify uses the modern css vars, especially for coloring html elements.

### CODE REVIEW

**Review by jGonzalez-CCPO — CHANGES_REQUESTED** [review]
Be aware that your changes are impacting other pages, like in /dashboard. the yellow square box now is no visible. **I know, we don't want to style an alert**

The new route is in the root folder so it will make it difficult to have tester pages similar to this in the future. **we may not have those tester pages in the future**

The combo box and fields are behaving correctly.

Icons are not part of the approved list.  **yes, they were just put there to ensure they can be colored successfully**

List of icons: 

https://www.figma.com/file/DjhHuxwq6eistDUyc6CKAh/*ATAT-Design-System-(Library)-(Master-%40-7b44558)?node-id=0%3A3352

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -129,7 +129,7 @@ export default {
 </script>
 
 <style scoped>
-.profile-card {
+/* .profile-card {
```
[the remark this answers is not in the document]
Comment (tonysumner-ccpo) [reply src/views/Dashboard.vue:132]: it's coming out on the task I'm working now.

File: src/sass/atat.scss [hunk 2]
```diff
@@ -1,5 +1,3 @@
-@import "variables";
```
[the remark this answers is not in the document]
Comment (tonysumner-ccpo) [reply src/sass/atat.scss:1]: src\sass\variables.scss

File: src/sass/variables.scss [hunk 3]
```diff
@@ -21,8 +21,8 @@ $btn-sizes: (
   'large': 54
 );
 
-// variables for colors
-$cyan : #05bfe7; // exact cyan color on atat
```
[the remark this answers is not in the document]
Comment (tonysumner-ccpo) [reply src/sass/variables.scss:25]: I removed it and put it in the 'src\plugins\theme.ts' that vuetify uses where our other theme colors are.

File: src/router/index.ts [hunk 4]
```diff
@@ -26,10 +26,10 @@ const routes: Array<RouteConfig> = [
     component: Dashboard,
   },
   {
-    path: "/sample/style",
```
[the remark this answers is not in the document]
Comment (tonysumner-ccpo) [reply src/router/index.ts:29]: yes, let's make it later if necessary.

File: src/sass/variables.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (jGonzalez-CCPO) [reply src/sass/variables.scss:25]: It looks like that color change was not committed on the theme.ts

File: src/router/index.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (jGonzalez-CCPO) [reply src/router/index.ts:29]: I was talking about making it only available in development or testing mode. 

The mother route is better to be defined from the very beginning, we can not use  `test` as a name for the folder so kyle and I came with the idea of naming it sample. can you please make your testing route in a subfolder like for example: 

sample/formFields

and not 

formFields/


You can also choice another name for the mother folder if you don't like sample/

**Review by jGonzalez-CCPO — CHANGES_REQUESTED** [review]

**Review by jGonzalez-CCPO — APPROVED** [review]

**Pull request merged**